packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -235,7 +227,7 @@ function UserMenu({
 
   const [open, setOpen] = useState(false);
 
-  const name = `${firstName} ${lastName}`;
+  const name = account ? `${account?.firstName} ${account?.lastName}` : '';
```

Comment:
Hmm... The way this is written, it looks like the output could be something like "Dave undefined" if one is defined and the other isn't.

Reply:
👍